scripts/initProject.pl

```diff
@@ -15,20 +15,21 @@
 
 
 my $CONFIG = <<ENDL;
-export REF = b37
-DUP_TYPE = markdup
+---
+ref: b37
+dup_type: markdup
 
-#TARGETS_FILE = intervals.bed
-#GENES_FILE = genes.txt
-EXOME = true
-CHR1_BASE_RECAL = true
+# targets_file: intervals.bed
+exome: true
+chr1_base_recal: true
```

Comment:
Maybe namespace these configurations by module name so it's more clear what they are used for